reppy/exceptions.py

```diff
@@ -36,3 +36,22 @@ def __init__(self, *args, **kwargs):
             self.status = args[1]
         ReppyException.__init__(self, *args, **kwargs)
 
+class SSLException(ReppyException):
+    def __init__(self, *args, **kwargs):
```

Comment:
Ok, just this one because the message is really clear or all of the new exceptions?